x/liquidstakeibc/keeper/msg_server.go

```diff
@@ -219,6 +217,20 @@ func (k msgServer) UpdateHostChain(
 			}
 			//fee limits validated in msg.ValidateBasic()
 			hc.Params.UnstakeFee = fee
+		case types.KeyLSMValidatorCap:
```

Comment:
Shouldn't it be part of an ICQ query? Who is updating these values?

Reply:
we can't issue an ICQ with a proof for the staking module params until `gaia` updates to v47

meanwhile, these params are going to be set manually
it is a different scenario as with the validator data since there can't be any beneficial outcome for us by maliciously changing them, here are the two cases:
- we set them higher than the value, which will cause delegations to fail
- we set them lower than the value, which will cap us in terms of how many delegations we can process

when `gaia` is on v47 we will update this and send an ICQ